patch/node/node_wrapper.h

```diff
@@ -51,6 +50,7 @@ struct _JSValue {
 };
 
 typedef struct _JSValue JS_Value;
+typedef struct _JSValue JS_Result;
```

Comment:
sure. I can revert this part. I was thinking, having JS_Result(JXResult) was helping in terms of convention to understand the code better. I may be wrong.

---
In reply to: [73199692](https://github.com/openT2T/node-apphost/pull/7#discussion_r73199692) [](ancestors = 73199692,73125188)